app/scripts/filters/util.js

```diff
@@ -348,30 +348,6 @@ angular.module('openshiftConsole')
           .replace(/^./, function(str){ return str.toUpperCase(); });
     };
   })
-  // Resource is either a resource object, or a name.  If resource is a name, kind and namespace must be specified
-  // Note that builds and deployments can only have their URL built correctly (including their config in the URL)
-  // if resource is an object
-  .filter('navigateResourceURL', function(Navigate) {
```

Comment:
We were accumulating enough navigation filters that I moved them to a separate file.